tree_performance/benchmarks/R.py

```diff
@@ -0,0 +1,41 @@
+import subprocess
+import pathlib
+import tskit
+import util
+
+VECTORISED = False
```

Comment:
The R implemention is vectorised I think

Reply:
I didn't think it was as it doesn't take a `chunk_size` as an argument - but maybe that's because it determines that itself?

Reply:
Probably should, but I couldn't figure out how to make R do it. It is definitely doing multiple sites at a time under the hood anyway though, this is one of the reasons it's fast (and the reason I started looking at vectorising in the first place)

Reply:
I've changed the flag to `True`.